.env.example

```
@@ -42,8 +42,8 @@ SUPABASE_USER_REPO_ENABLED=false
 SUPABASE_API_KEYS_ENABLED=false
 SUPABASE_BILLING_STORE_ENABLED=false
 
-# Optional Langfuse tracing
-LANGFUSE_ENABLED=false
-LANGFUSE_BASE_URL=https://cloud.langfuse.com
-LANGFUSE_PUBLIC_KEY=
-LANGFUSE_SECRET_KEY=
+# Local Langfuse tracing
+LANGFUSE_ENABLED=true
+LANGFUSE_BASE_URL=http://langfuse:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFUSE_BASE_URL key should go before the LANGFUSE_ENABLED key

(UnorderedKey)